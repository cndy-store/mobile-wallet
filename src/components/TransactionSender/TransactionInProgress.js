import React, { Component } from 'react';
import PropTypes from 'prop-types';
import {
  Button,
  StatusBar,
  StyleSheet,
  View,
  Text,
  TextInput
} from 'react-native';
import Dimensions from 'Dimensions';

const { width, height } = Dimensions.get('window');

class TransactionInProgress extends Component {
  render() {
    return (
      <View>
        <Text>In Progress...</Text>
      </View>
    );
  }
}

TransactionInProgress.propTypes = {};

export default TransactionInProgress;
