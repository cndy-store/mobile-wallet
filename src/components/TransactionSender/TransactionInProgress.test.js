import React from 'react';
import { render } from '../../__tests__/renderer';
import TransactionInProgress from './TransactionInProgress';

it('renders correctly without an error', () => {
  const { toJSON } = render(<TransactionInProgress />);
  expect(toJSON()).toMatchSnapshot();
});
