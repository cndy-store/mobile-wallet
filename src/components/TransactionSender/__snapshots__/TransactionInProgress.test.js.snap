// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly without an error 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "height": 1067.2,
      "justifyContent": "center",
    }
  }
>
  <View
    style={Object {}}
  >
    <View
      collapsable={undefined}
      style={
        Object {
          "marginLeft": 60,
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        }
      }
    >
      <Image
        collapsable={undefined}
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../assets/img/food/1.png",
          }
        }
        style={
          Object {
            "height": 90,
            "transform": Array [
              Object {
                "rotate": "0deg",
              },
              Object {
                "translateY": 5,
              },
            ],
            "width": 150,
          }
        }
      />
    </View>
    <View
      collapsable={undefined}
      style={
        Object {
          "marginLeft": 0,
          "marginTop": -100,
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        }
      }
    >
      <Image
        collapsable={undefined}
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../assets/img/food/2.png",
          }
        }
        style={
          Object {
            "height": 90,
            "transform": Array [
              Object {
                "rotate": "0deg",
              },
              Object {
                "translateY": 5,
              },
            ],
            "width": 150,
          }
        }
      />
    </View>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#000",
        "fontFamily": "System",
        "fontSize": 16,
      }
    }
    uppercase={false}
    virtual={undefined}
  >
    Transaction in Progress...
  </Text>
</View>
`;
